tweepy/user.py

```diff
@@ -63,6 +63,8 @@ class User(HashableID, DataMapping):
         The URL specified in the user's profile, if present.
     verified : bool | None
         Indicates if this user is a verified Twitter User.
+    verified_type : "blue" | "business" | "government" | "none"
```

Comment:
This should be the type, not the literal strings it could be, as there could easily be new and/or undocumented verified types. This could also be `None` when the field isn't in the data.

```suggestion
    verified_type : str | None
```